modules/history/src/views/full/index.jsx

```diff
@@ -6,6 +6,7 @@ import { MessageViewer } from './MessageViewer'
 import { ConversationPicker } from './ConversationPicker'
 
 const CONV_PARAM_NAME = 'conversation'
+const BASE_API_PATH = '/mod/history/'
```

Comment:
That's not something which is bound to change, i'd suggest keeping the full url in the paths when you make your api calls. It's shorter and more clean